portfolio/src/main/java/com/google/sps/servlets/UserServlet.java

```diff
@@ -22,18 +24,28 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that displays login status of user. */
+/** Servlet that retrieves user login status. */
 @WebServlet("/user-login")
 public class UserServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html");
     UserService userService = UserServiceFactory.getUserService();
-    if (userService.isUserLoggedIn()) {
-      response.getWriter().println("<p>User logged in</p>");
+    
+    Gson gson = new Gson();
+    if(userService.isUserLoggedIn()) {
+      String urlToRedirectToAfterUserLogsOut = "/html/comments.html";
+      String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
+
+      User newUser = User.create(true, "", logoutUrl);
```

Comment:
Can you add inline comments for true and "".

Reply:
done